HardwareCheckout/__main__.py

```diff
@@ -1,8 +1,10 @@
 import tornado
+import tornado.options
 
 from . import create_app, create_redirect
 from .config import ssl_config
 
+tornado.options.parse_command_line()
```

Comment:
Is this being used? I don't see where/how it is affecting anything

Reply:
https://www.tornadoweb.org/en/stable/options.html ; calling this gets tornado to parse `sys.argv` . I did it so that I could do `python3 -m HardwareCheckout --logging=debug`